cristin-organization/src/main/java/no/unit/nva/cristin/organization/IFetchApiClient.java

```diff
@@ -0,0 +1,9 @@
+package no.unit.nva.cristin.organization;
+
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+
+public interface IFetchApiClient<T, R> {
+
+    R executeFetch(T params) throws ApiGatewayException;
```

Comment:
Should i rather use a Map as param type?

Reply:
If it always is a map, use a map.

Reply:
T is fine, you are then free to choose implementation later.